src/sentry/static/sentry/app/views/eventsV2/miniGraph.tsx

```diff
@@ -124,7 +125,24 @@ class MiniGraph extends React.Component<Props> {
                 },
               }}
               yAxis={{
-                show: false,
+                show: true,
+                axisLine: {
+                  show: false,
+                },
+                axisLabel: {
+                  color: theme.gray400,
+                  fontFamily: theme.text.family,
+                  fontSize: 12,
+                  formatter: formatAbbreviatedNumber,
+                  inside: true,
+                  showMinLabel: false,
+                  showMaxLabel: false,
+                },
+                splitNumber: 3,
+                splitLine: {
+                  show: false,
+                },
+                zlevel: 2147483647,
```

Comment:
This could get funny if there are any modals that show up on this page, or if a dropdown from the global selection header crosses these charts. You could use theme.zIndex.header or add another alias for a value less than max int.

Reply:
Fixed in 8035ffb